capsules/extra/src/app_loader.rs

```diff
@@ -106,7 +106,7 @@ mod ro_allow {
     pub const COUNT: u8 = 1;
 }
 
-pub const BUF_LEN: usize = 512;
+pub const BUF_LEN: usize = 4096;
```

Comment:
Can you add a comment here explaining why this number is chosen [can just include a link to this PR] so that future readers are aware of the performance significance?